### PR TITLE
fix: Add dummy install plan to oma-data-gov

### PR DESCRIPTION
**Pull request by JimHagan**
# Summary

At the recommendation from some people like Stijn and others in `help-newreic-io` I've created a `dummy` install plan in order to allow users to have relatively effortless dashboard install.

## Pre merge checklist

- [✅ ] Did you check you NRQL syntax? - Does it work?
- [ ✅] Did you check your dashboard image quality? -  Do they look good?
- [✅ ] Did you check that your alerts actually work?
- [ ✅] Did you include an InstallPlan and Documentation reference?
- [✅ ] Did you check your descriptive content for voice, tone, spelling and grammar errors?
- [ ✅] Did you attach images of your dashboards to the PR so we can see them working?

### Screenshots

Attach images of any visual changes, such as a dashboard here.

### CODE REVIEW

File: install/solutions-hub/dashboards/install.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+id: solutions-hub-dashboards
+name: solutions-hub
+title: New Relic Solutions Hub Dashboards
+description: New Relic's solution framework offers a repeatable and proven approach to help you do just that. Using proven best practices and expert guidance, your organization can move faster, stay on course, and transform successfully.
+  
+
+target:
```
Comment (jpvajda) [inline install/solutions-hub/dashboards/install.yml:7]: I'm not sure if the unknown value will cause an issue here. Was this what you were instructed to do here @JimHagan ?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: install/solutions-hub/dashboards/install.yml (hunk 1 above)
Comment (jpvajda) [reply install/solutions-hub/dashboards/install.yml:7]: We confirmed with @jbeveland27 what will occur in the UI in this case, and though it's not ideal it did identify a potential improvement for us to look at.

**Review by jpvajda — APPROVED** [review]

**Comment by jpvajda** [comment]
This is approved now, but  I wanted to double check with @JimHagan if he's ok with the current UI experience that would show the user this type of unknown, unknown message in the guided install UI.

Screen Shot 2022-02-07 at 10 27 06 AM

**Comment by JimHagan** [comment]
@jpvajda this would be a decent compromise experience at present.  Thanks.

**Pull request merged**